### ISSUE TITLE
Flagging impossible due to scrollbar

### ISSUE THREAD

**Issue by vibs29** [roundcube/roundcubemail#9643]
### Prerequisites

- [X] I have [searched](https://github.com/roundcube/roundcubemail/issues?q=is%3Aissue) for duplicate or closed issues
- [X] I can recreate the issue with all plugins disabled

### Describe the issue

The middle pane (message list) in threaded view has a button to each message's right side that is shaped like a star. I believe clicking it will "flag" the message. However as soon as the mouse reaches that star button the hidden scrollbar pops into view covering the star button. So it's impossible to click the star button.

Debian Linux (bookworm), LXDE

### What browser(s) are you seeing the problem on?

Firefox

### What version of PHP are you using?

8.2.20

### What version of Roundcube are you using?

1.6.5

**Comment by alecpl** [comment]
There's no stars in the Elastic skin. So, what skin? What plugins? What browser? Could you provide a screenshot?

**Comment by vibs29** [comment]
You're right, sorry. It's in the classic skin. No plugins. The browser is the one I listed.

I run with page at 80% zoom. I can get the mouse over maybe one pixel of the star's left side, but any further brings the scrollbar into view obscuring the star. At 100% zoom, I can get the mouse over a few more pixels before the scrollbar appears and obscures it. I'd say that if there is any part of the star that my mouse can't reach because the scrollbar appears, then that's a bug. By that measure, even the elastic skin with page at 100% zoom has at least one pixel on flag's right that is unreachable by mouse due to scrollbar's appearing instead.
